src/PokemonGymOwner.java

```diff
@@ -0,0 +1,39 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class PokemonGymOwner {
```

Comment:
PokemonGymOwner is basically a PokemonTrainer, but a gymowner also has a `town`. The `name` and `pokemons` attributes Both classes have, so if you let PokemonGymOwner inherit from PokemonTrainer, you don't have to define a `name` and `pokemons` in this class, but instead can use them from the super (PokemonTrainer).